playwright-tests/playwright.config.ts

```diff
@@ -27,7 +27,7 @@ const config: PlaywrightTestConfig = {
   workers: 1,
   reporter: [
     ['github'],
-    ['line'],
+    ['list'],
```

Comment:
Why do we want to do this change?

Reply:
this brings emoji checkbox in the console output where the test is called. Pretty good to see where the failed test starts, please see  linked screenshot: https://www.screencast.com/t/usqwKCSC1B